src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public Task[] tasks;
+
+
+    public static void main(String[] args) throws DukeException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("___________________________________");
+        System.out.println("Hello! I'm Duke\n What can I do for you?");
+        System.out.println("___________________________________");
+
+        Task[] tasks = new Task[100];
+        int counter = 0;
+        ArrayList<Task> collection = new ArrayList<>();
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+
+
+
+        while (!input.equals("bye")) {
+            if (input.equals("list")) {
+                System.out.println("___________________________________");
+                for (int i = 0; i < counter; i++) {
+                    System.out.println( (i+1) + "."
```

Comment:
Seems like there's an extra whitespace on this line, might need to remove it to follow the coding standards